### PR TITLE
Improve cmd

### PR DESCRIPTION
**Pull request by wsdjeg**

### CODE REVIEW

File: docs/cn/install.cmd [hunk 1]
```diff
@@ -20,81 +20,92 @@ $repo_url    = "https://github.com/SpaceVim/SpaceVim.git"
...
 sleep 1
 
 
-echo "==> Testing git"
+echo "==> 测试 git 命令"
 if (Get-Command "git" -ErrorAction SilentlyContinue) {
-    git version
-    echo "[OK] Test successfully. Moving to next..."
+  git version
+    echo "[OK] 测试成功. 开始下一个测试..."
```
Comment (Gabirel) [inline docs/cn/install.cmd]: Please make sure they have the same indent.

Comment (Gabirel) [reply docs/cn/install.cmd]: And it applies to the following lines

File: docs/cn/install.cmd [hunk 2]
```diff
@@ -20,81 +20,92 @@ $repo_url    = "https://github.com/SpaceVim/SpaceVim.git"
...
+echo "==> 测试 gvim 命令"
 if (Get-Command "gvim" -ErrorAction SilentlyContinue) {
-    echo ($(vim --version) -split '\n')[0]
-    echo "[OK] Test successfully. Moving to next..."
+  echo ($(vim --version) -split '\n')[0]
+    echo "[OK] 测试成功. 开始下一个测试..."
     sleep 1
 } else {
-    echo "[WARNING] Unable to find 'gvim.exe' in your PATH. But intallation still can continue..."
+  echo "[WARNING] 无法在你的 PATH 中发现 'gvim.exe' 命令. 安装已继续..."
```
Comment (Gabirel) [inline docs/cn/install.cmd]: “安装已继续” ==> "但，安装仍可继续"

**Review by Gabirel — CHANGES_REQUESTED** [review]

File: docs/cn/install.cmd (hunk 2 above)
Comment (wsdjeg) [reply docs/cn/install.cmd]: @Gabirel 你知不知道，cmd 里面怎么彩色输出呀，我想整个脚本流程及界面和 bash 一样的

Comment (Gabirel) [reply docs/cn/install.cmd]: No, I'm not familiar with powershell

**Pull request merged**